src/main/java/org/sonarlint/intellij/notifications/SonarLintProjectNotifications.java

```diff
@@ -108,17 +108,21 @@ public void notifyProjectStorageInvalid() {
     shown = true;
   }
 
-  public void notifyWiderLanguageSupport(String language) {
+  public void notifyLanguagePromotion(String content) {
     var notification = IN_CONTEXT_PROMOTION_GROUP.createNotification(
       TITLE_SONARLINT_SUGGESTIONS,
-      "Enable " + language +" analysis by connecting your project",
-      NotificationType.WARNING);
+      content,
+      NotificationType.INFORMATION);
+    addActions(notification);
+    notification.notify(myProject);
+  }
+
+  private static void addActions(Notification notification) {
```

Comment:
This separate method makes less sense now as it will be called only once. You can also keep it, as you prefer